Integrationtests fail when Python path contains space

With a Python installation at `C:\Program Files\Python35\python.exe`, integrationtests fail in  TestPrecompiledHeaders.testSampleproject

```
c1xx: fatal error C1083: Cannot open source file: 'doesnotexist.cpp': No such fi
le or directory
clcache: preprocessor failed
.       C:\Program Files\Python35\python.exe C:\workspace\clcache\clcache.py /no
logo /EHsc /c /W3 /Gs /MT /Ycstable.h applib.cpp
Der Befehl "C:\Program" ist entweder falsch geschrieben oder
konnte nicht gefunden werden.
NMAKE : fatal error U1077: 'C:\Program' : return code '0x1'
Stop.
Eminimal.cpp
#line 1 "tests\\integrationtests\\minimal.cpp"
void f() {}
minimal.cpp
```

This is because CLCACHE_CMD is joined using spaces, so the Python path is not quoted in the shell.

This is not important for me right now. Just wanted to document it.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,7 +3,8 @@ clcache changelog
 
 ## Upcoming release
 
- * Nothing yet!
+ * Internal: Fixed running integration tests when the clcache source code is
+   stored in a path with spaces (GH #206).
 
 ## clcache 3.3.0 (2016-09-07)
 

--- a/integrationtests.py
+++ b/integrationtests.py
@@ -559,7 +559,7 @@ class TestHits(unittest.TestCase):
 class TestPrecompiledHeaders(unittest.TestCase):
     def testSampleproject(self):
         with cd(os.path.join(ASSETS_DIR, "precompiled-headers")):
-            cpp = ' '.join(CLCACHE_CMD)
+            cpp = subprocess.list2cmdline(CLCACHE_CMD)
 
             testEnvironment = dict(os.environ, CPP=cpp)
 

